src/app/login/login.hbs

```diff
@@ -0,0 +1,17 @@
+<div class="login-container">
+    <div class="login-text-box">
+        <h3>Login</h3>
+        <p style="font-size: 80%;">Get access to your Orders, Wishlist and Recommendations</p>
+    </div>
+    <form class="login-form">
+        <div class="wrap-input100">
+            <input class="input100" type="text" name="email">
+            <span class="focus-input100" data-placeholder="Email"></span>
+        </div>
+        <div class="wrap-input100">
+            <input class="input100" type="password" name="pass">
+            <span class="focus-input100" data-placeholder="Password"></span>
+        </div>
+        <a class="btn-login" href="#" onclick="clickMe()">Login</a>
```

Comment:
An <a> tag for submit button ??? are you kidding me?? Where did you learn that?

Reply:
It wasn't intentional! Apologies for doing such a thing.